<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowseWindow</class>
 <widget class="QMainWindow" name="BrowseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(18, 18, 18)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="Header" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="Logo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../build-Foxlike-Desktop_Qt_6_6_0_MinGW_64_bit-Debug/resources/uielements/Logo.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="HeaderButtons" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="search">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	background-color: rgb(32, 32, 32);
	border-radius: 20%;
	color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
            <property name="placeholderText">
             <string>ðŸ”Ž Search</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browseButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: transparent;
	color: rgb(150, 150, 150);
}

QPushButton:hover  {
	background-color: transparent;
	color: rgb(255, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>All games</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="filterButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: transparent;
	color: rgb(150, 150, 150);
}

QPushButton:hover  {
	background-color: transparent;
	color: rgb(255, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="accountButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../build-Foxlike-Desktop_Qt_6_6_0_MinGW_64_bit-Debug/resources/uielements/avatar-icon.png</normaloff>../build-Foxlike-Desktop_Qt_6_6_0_MinGW_64_bit-Debug/resources/uielements/avatar-icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="filterButtons" native="true">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="6">
        <widget class="QLabel" name="priceText">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Any price</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="genreComboBox">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
    border: 0px;
    background: rgb(18, 18, 18);
    color: #fff;
    padding: 5px;
    border-radius: 5px;
    selection-background-color: #555;
}

QComboBox::drop-down {
    width: 20px;
    border: none;
}

QComboBox QAbstractItemView {
    border: 1px solid #555;
    background: #333;
    color: #fff;
    selection-background-color: #555;
}

QComboBox::item {
    padding: 5px;
}

QComboBox::item:selected {
    background: #555;
}

QComboBox::item:hover {
    background: #444;
}

QComboBox::disabled {
    color: #777;
    background: #333;
}
</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="7">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="genreLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Genre:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QSlider" name="priceSlider">
         <property name="styleSheet">
          <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #555;
    background: #333;
    height: 10px;
    border-radius: 5px;
}

QSlider::handle:horizontal {
    background: rgb(32, 32, 32);
    width: 14px;
    margin: -2px 0;
    border-radius: 7px;
}

QSlider::sub-page:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #004D40, stop:1 #00695C);
    border: 1px solid #000;
    height: 10px;
    border-radius: 5px;
}

QSlider::add-page:horizontal {
    background: #333;
    border: 1px solid #000;
    height: 10px;
    border-radius: 5px;
}

QSlider::add-page:horizontal:disabled,
QSlider::sub-page:horizontal:disabled,
QSlider::handle:horizontal:disabled {
    background: #555;
    border-color: #555;
}

QSlider::add-page:horizontal:disabled {
    border-radius: 5px;
}

QSlider::sub-page:horizontal:disabled {
    border-radius: 5px;
}

QSlider::handle:horizontal:disabled {
    border-radius: 9px;
}</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="priceLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Price:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true">QWidget {
	background-color: rgb(18, 18, 18)
}

 QScrollBar:vertical {
    background: rgb(22, 22, 22);
    width: 14px;
	border-radius: 0px;
 }

QScrollBar::handle:vertical {	
	background-color: rgb(28, 28, 28);
	min-height: 30px;
	border-radius: 7px;
}

QScrollBar::sub-line:vertical {
	height: 0px;
}

QScrollBar::handle:vertical:hover{	
	background-color: rgb(32, 32, 32);
}
QScrollBar::handle:vertical:pressed {	
	background-color: rgb(32, 32, 32);
}

QScrollBar::add-line:vertical {
	height: 0px;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1149</width>
            <height>573</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QGridLayout" name="GamesGrid"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1277</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
